tests/Agent/UnitTests/CompositeTests/CompositeTestAgent.cs

```diff
@@ -405,6 +412,45 @@ public void SetEventListenerSamplersEnabled(bool enable)
             CurrentConfiguration.EventListenerSamplersEnabled = enable;
         }
 
+        public void StartActivity()
+        {
+            _defaultActivityIdFormat = Activity.DefaultIdFormat;
+            Activity.DefaultIdFormat = ActivityIdFormat.W3C;
+            _activitySource = new ActivitySource("CompositeTestAgent");
```

Comment:
The activitysource should only need to be created once, and we can probably just dispose it when the compositetestagent is disposed. However, the activity creation and and settings for the activity such as the default id format, should probably be pushed into the test classes so that it will be easier to manage the lifetimes of those activities with the using statement or block.

Reply:
No longer relevant since I reverted the composite test changes